ethereum.md

```diff
@@ -182,14 +196,14 @@ To do so, we'll extend sort `JSON` with some IELE specific syntax, and provide a
          </message>
 
     rule <k> #adjustGas => . ... </k> <checkGas> true </checkGas>
+    rule STATUS:Int ~> #adjustGas => #load %0 STATUS ~> #adjustGas
 
     syntax IELECommand ::= "#finishTx"
  // ----------------------------------
-    rule <k> #exception _ ~> #finishTx => #popCallStack ~> #popWorldState ~> #popSubstate ... </k>
-    rule <k> #revert _    ~> #finishTx => #popCallStack ~> #popWorldState ~> #popSubstate ~> #refund GAVAIL ... </k> <gas> GAVAIL </gas>       
+    rule <k> #exception STATUS ~> #finishTx => #popCallStack ~> #popWorldState ~> #popSubstate ~> #load %0 STATUS ... </k>
```

Comment:
Is this change (loading STATUS to the register) not listed in the PR summary?

Reply:
updated